README.md

```diff
@@ -0,0 +1,89 @@
+- [Formik](https://github.com/jaredpalmer/formik) w/ [yup](https://github.com/jquense/yup) for forms
+
+Backend:
+
+- [Express](https://expressjs.com/) as our web framework w/ [Apollo Server](https://github.com/apollographql/apollo-server) as our GraphQL server
+- [Sequelize](http://docs.sequelizejs.com/) as our ORM w/ [MySQL](https://www.mysql.com/) as our database
+- [Passport](http://www.passportjs.org/) w/ [Passport SAML](https://github.com/bergie/passport-saml) for external user authentication
+- [JWT](https://jwt.io/) for API authentication & short lived tokens
+- [Multer](https://github.com/expressjs/multer) for file uploads
+
+See each of their `package.json` files for the other tools we used (eg. transpiling, linting, testing).
+
+### New to React, Redux, & GraphQL?
+
+Here are some useful resources on these tools and our architecture:
+
+#### React
+
+- [The Beginner's Guide to ReactJS](https://egghead.io/courses/the-beginner-s-guide-to-reactjs) by Kent C. Dodds
+- [the Road to learn React](https://roadtoreact.com/)
+- [React Express](http://www.react.express/)
+- [awesome-react](https://github.com/enaqx/awesome-react)
+- [react-redux-links](https://github.com/markerikson/react-redux-links) by Mark Erikson
+- [Presentational and Container Components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) by Dan Abramov
+- [Simple React Patterns](http://lucasmreis.github.io/blog/simple-react-patterns/) by Lucas Reis
+- [Fullstack React](https://www.fullstackreact.com/)
+- [React for Beginners](https://reactforbeginners.com/) by Wes Bos
+- [Advanced React](https://courses.totalreact.com/p/advanced-react-free) by Ryan Florence
+
+#### Redux
+
+- [Getting Started with Redux](https://egghead.io/courses/getting-started-with-redux) by Dan Abramov
+- [Building React Applications with Idiomatic Redux](https://egghead.io/courses/building-react-applications-with-idiomatic-redux) by Dan Abramov
+- [A cartoon intro to Redux](https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6) by Lin Clark
+- [Live React: Hot Reloading with Time Travel](https://www.youtube.com/watch?v=xsSnOQynTHs) by Dan Abramov
+- [10 Tips for Better Redux Architecture](https://medium.com/javascript-scene/10-tips-for-better-redux-architecture-69250425af44) by Eric Elliott
+- [Learn Redux](https://learnredux.com/) by Wes Bos
+
+#### GraphQL
+
+- [GraphQL](http://graphql.org/)
+- [Apollo GraphQL](https://www.apollographql.com/)
+- [Apollo Developers Blog](https://dev-blog.apollodata.com/)
+- [The GitHub GraphQL API](https://githubengineering.com/the-github-graphql-api/)
+- [From REST to GraphQL](https://0x2a.sh/from-rest-to-graphql-b4e95e94c26b) by Jacob Gillespie
+- [awesome-graphql](https://github.com/chentsulin/awesome-graphql)
+
+## Development
+
+You'll need to be running both the frontend and backend for development. Check out each of their README's for further instructions.
```

Comment:
I would probably reiterate the instructions here or link to the front end read me and then backend readme

Reply:
I'd lean towards linking.

Reply:
I'm going to say neither. You're in the top level of the repo, it's not hard to click on the  `backend` or `frontend` folders and read their README. Additionally, links inside our own repo will require maintenance – eg. when we move the repo to the org, we'll have to update the links.